src/dateutils.spec.js

```diff
@@ -20,6 +20,46 @@ describe('dateutils', function () {
     });
   });
 
+  describe('sameWeek tests', () => {
+    it('Expect sameWeek to return true, for two days on the same week ', () => {
+      const {sameWeek} = dateutils;
+      const a = '2021-01-05';
+      const b = '2021-01-06';
+      expect(sameWeek(a, b, 1)).toBe(true);
+    });
+
+    it('Expect sameWeek to return true, for two days on the same week ', () => {
```

Comment:
Why do we need two tests for the same sanerio?

Reply:
It should of being this test: `expect(sameWeek(prevDate, date, 1)).toBe(true);`
(first date passed is after the second date passed)